tests/common/test_transaction.py

```diff
@@ -94,13 +98,17 @@ def test_output_serialization(user_Ed25519, user_pub):
 
 
 def test_output_deserialization(user_Ed25519, user_pub):
+    import base58
```

Comment:
This could be imported at the top since `base58` is not under test, and is a third party library that we assume to be stable.